static/timing-info-widget.js

```diff
@@ -176,76 +138,87 @@ TimingInfo.prototype.initializeChartDataFromResult = function (compileResult, to
     if (compileResult.didExecute) {
         if (compileResult.buildResult) {
             if (compileResult.buildResult.packageDownloadAndUnzipTime) {
-                timings.push({
-                    step: 'Download binary from cache',
-                    time: compileResult.buildResult.packageDownloadAndUnzipTime,
-                });
+                pushTimingInfo(data,
+                    'Download binary from cache',
+                    compileResult.buildResult.packageDownloadAndUnzipTime);
             } else {
-                if (compileResult.execResult.buildResult) {
-                    this.addBuildResultToTimings(timings, compileResult.execResult.buildResult);
+                if (compileResult.execResult && compileResult.execResult.buildResult) {
```

Comment:
@partouf can you check that this is correct? I hit a weird case where this failed when refreshing the page because execResult was null in the executor pane (And resending the compilation made it work)

Reply:
I think this makes sense yes, might fix a bug I've seen before

Reply:
hmm, but then this would be in the wrong else

Reply:
Should probably be at same level as `if (compileResult.buildResult) {` - if that's not true, then do `} else if (compileResult.execResult && compileResult.execResult.buildResult) { `